.gitignore

```diff
@@ -1,2 +1,5 @@
 node_modules
 .env
+.DS_Store
+README.html
```

Comment:
convenience for when I check my `README.md` updates & rendering

Reply:
Cool, I was reading about this the other day: https://help.github.com/articles/ignoring-files/#explicit-repository-excludes

Looks like you can possibly just add your <kbd>README.html</kbd> to your own local <kbd>.git/info/exclude</kbd> blocklist since it is a testing artifact on your local machine only.

Reply:
Yes, I was going to suggest that as well: please include these in your global ignore list.